content/docs/v1.11.0/docs/maintainers/release.md

```diff
@@ -23,7 +23,7 @@ release. We use `<TAG>` as a placeholder for the release tag (e.g. `v1.4.0`).
      message must be *exactly* `"Update CHANGELOG for <TAG> release"`, as a bot
      will look for this commit and cherry-pick it to update the main branch
      (starting with Antrea v1.0). The
-     [process-changelog.go](../../hack/release/process-changelog.go) script may
+     [process-changelog.go](https://github.com/antrea-io/antrea/blob/v1.11.0/hack/release/process-changelog.go) script may
      be used to easily generate links to PRs and the Github profiles of PR
      authors.
    - a commit to update [VERSION](../../VERSION) as needed, using the following
```

Comment:
@antoninbas could you please help to find why this is not getting translated. I have added unit test for the same, it gets translated there.
Same issue at L85

Reply:
Please ignore, I didn't run `make` command after making changes.